src/lib/party.ts

```diff
@@ -740,7 +751,15 @@ export class Party<P> {
     return this.body.publicKey;
   }
 
-  static createUnsigned(publicKey: PublicKey): Party<void> {
+  static defaultParty(address: PublicKey) {
+    const body = Body.keepAll(address);
+    const predicate = AccountPredicate.ignoreAll();
+    return new Party(body, predicate) as Party & {
```

Comment:
We are using this union type `Party & { predicate }` a decent amount. Would it be helpful to write this as a type with a name that self documents it's purpose?